universal.py

```diff
@@ -24,6 +25,34 @@
 middlewares = [MiddlewareTimeout(), MiddlewareRatelimit(), MiddlewareBaiduCloud(), MiddlewareConcurrentLock()]
 
 
+def get_bot_info():
```

Comment:
I think maybe we need some kind of registry.  
During initialization, we register all kind of LLM using something like:
`llmRegistry.register('chatgpt-web', ChatGPTWebAdapter)`

and then we can query all of available llms using:

`llmRegistry.list())`